XLForm/XL/Cell/XLFormBaseCell.h

```diff
@@ -35,7 +35,7 @@
 
 @property (nonatomic, weak) XLFormRowDescriptor * rowDescriptor;
 
--(XLFormViewController *)formViewController;
+@property (nonatomic, readonly, strong) XLFormViewController *formViewController;
```

Comment:
@jkereako thanks for the doc reference. 
I think you can even use dot-syntax with methods without parameters. At least i have been doing it so far all over the base code. 
Regarding the property vs methods, for these particular cases i mentioned i think is more clear to expose them as a method otherwise i have to make sure or check i'm not implementing the accessor method. 
I mean, the method is more clear showing the real propose of it, i get confused  when i find a property that is not actually a property (Does the instance has a internal state to save it or not? ).

BTW strong doesn't apply to readonly properties without accessor method. I want to avoid this confusion and make it clear ;).